numpy/_core/tests/test_regression.py

```diff
@@ -427,18 +428,21 @@ def test_lexsort_zerolen_custom_strides(self):
         xs = np.array([], dtype='i8')
         assert np.lexsort((xs,)).shape[0] == 0  # Works
 
-        xs.strides = (16,)
+        with pytest.warns(DeprecationWarning):
+            xs.strides = (16,)
         assert np.lexsort((xs,)).shape[0] == 0  # Was: MemoryError
 
     def test_lexsort_zerolen_custom_strides_2d(self):
         xs = np.array([], dtype='i8')
 
         xs.shape = (0, 2)
-        xs.strides = (16, 16)
+        with pytest.warns(DeprecationWarning):
+            xs.strides = (16, 16)
         assert np.lexsort((xs,), axis=0).shape[0] == 0
 
         xs.shape = (2, 0)
-        xs.strides = (16, 16)
+        with pytest.warns(DeprecationWarning):
+            xs.strides = (16, 16)
```

Comment:
These ones would be nice to change, since the test is unrelated to the strides setting itself.